### PR TITLE
Add methods for grading periods and assignments

### LINKED ISSUE #28: Create Assignment
I was curious if it is possible to create an assignment?  I do not see anything in main.py and just wanted to make sure I wasn't missing something.  Thanks for the work on this, very useful!

### LINKED ISSUE #40: Please implement get_grading_periods and get_grading_period
I'll buy you a coffee if you can please implement get_grading_periods and get_grading_period.

### PR DESCRIPTION
**Pull request by CoderN-P**
Added:

- `get_grading_periods`
- `get_grading_period`
- `create_assignment`

Fixes #40
Fixes #28

### CODE REVIEW

**Review by ErikBoesen — CHANGES_REQUESTED** [review]
Whoops, forgot to hit submit on this!

File: schoolopy/main.py [hunk 1]
```diff
@@ -1901,6 +1901,8 @@ def delete_grading_group(self, group_id, section_id):
         """
         self._delete('sections/%s/grading_groups/%s' % (section_id, group_id))
 
+    def create_assignment(self, assignment, section_id):
+        return Assignment(self._post("/sections/%s/assignments" % section_id, assignment.json()))
```
[the remark this answers is not in the document]
Comment (CoderN-P) [reply schoolopy/main.py:1905]: Hi, thanks for replying. I will change to single quotes and test again later today.

[the remark this answers is not in the document]
Comment (ErikBoesen) [reply schoolopy/main.py]: Sounds good—I don't see any particular reason why it wouldn't work, after all.

**Comment by ErikBoesen** [comment]
Thank you so much for your contribution!

**Comment by CoderN-P** [comment]
Thank you for making this amazing lib too!

**Pull request merged**